project/CONTRIBUTING.md

```diff
@@ -180,9 +180,9 @@ namespace Foo;
 
 interface BarInterface
 {
-
-  // NEXT_MAJOR: Uncomment this method
   /**
+   * NEXT_MAJOR: Uncomment this method
```

Comment:
> the guy who will release the next major knows how to handle the NEXT_MAJOR tags

Or not. This is not the role of the releases guy to remove the NEXT_MAJOR comments.